doc/source/_templates/navbar-anyscale.html

```diff
@@ -1,4 +1,4 @@
-<a id="try-anyscale-href" href="https://forms.gle/DFin3AzKEDB19uXZ6">
+<a id="try-anyscale-href" href="https://www.anyscale.com">
```

Comment:
this line change is unrelated? could you rebase properly?

Reply:
ok

Reply:
https://github.com/ray-project/ray/pull/45016/files  from this?